staging/src/k8s.io/client-go/discovery/cached/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cached provides a discovery client for memcached.
+package cached // import "k8s.io/client-go/discovery/cached"
```

Comment:
do we need to add new `doc.go` for every dir? i thought just correcting the existing ones is fine.

Reply:
> Look through sub directories and see if there are any other important directories that are likely to be used by outside users and add a doc.go

As specified in #68231

Reply:
I would prefer uniformity with a doc.go everywhere.

Reply:
I added a doc.go in every directory that was not auto-generated in commit 2267ab3